apps/wpcom-block-editor/src/wpcom/features/tracking/delegate-event-tracking.js

```diff
@@ -79,6 +100,12 @@ export default ( event ) => {
 
 	matchingEvents.forEach( ( match ) => {
 		debug( 'triggering "%s". target: "%s"', match.event, match.target );
+		subscribers?.[ match.mapping.id ].before.forEach( ( subscriber ) =>
+			subscriber( match.mapping, match.event, match.target )
+		);
 		match.mapping.handler( match.event, match.target );
+		subscribers?.[ match.mapping.id ].after.forEach( ( subscriber ) =>
```

Comment:
I think on L 103 and 107 we need to move this `?` down a tier (ex `subscribers[match.mapping.id]?.before...`).  Subscribers should always exist since we define it as an empty object, however `subscriber[ someId ]` may not exist so calling `.before` can trigger an error breaking some of the event handling.

Ex - open the post editor and try to open the navigation sidebar.  This should trigger the `wpcom_block_editor_close_click` event, but instead errors:

![Screen Shot 2021-06-11 at 10 56 14 AM](https://user-images.githubusercontent.com/28742426/121706462-a617f500-caa3-11eb-8368-255160c79cff.png)

Reply:
Thanks! Fixed it!